package.json

```diff
@@ -76,25 +76,25 @@
     "zone.js": "~0.10.2"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.900.2",
-    "@angular-devkit/build-ng-packagr": "~0.900.2",
-    "@angular/cli": "^9.0.2",
-    "@angular/compiler": "^9.0.2",
-    "@angular/compiler-cli": "^9.0.2",
-    "@angular/language-service": "^9.0.2",
-    "@angular/platform-browser-dynamic": "^9.0.2",
-    "@angular/platform-server": "^9.0.2",
-    "@angular/router": "^9.0.2",
+    "@angular-devkit/build-angular": "^0.900.4",
+    "@angular-devkit/build-ng-packagr": "^0.900.4",
+    "@angular/cli": "^9.0.4",
+    "@angular/compiler": "^9.0.4",
+    "@angular/compiler-cli": "^9.0.4",
+    "@angular/language-service": "^9.0.4",
+    "@angular/platform-browser-dynamic": "^9.0.4",
+    "@angular/platform-server": "^9.0.4",
+    "@angular/router": "^9.0.4",
     "@commitlint/cli": "^8.2.0",
     "@nestjs/schematics": "^6.7.0",
-    "@nestjs/testing": "^6.8.3",
+    "@nestjs/testing": "^6.11.10",
```

Comment:
Please remove, we do not have nestjs in there any more, I must have missed it when removing the backend.

Reply:
I create a separate PR